### ISSUE TITLE
Breaking ReadTheDocs because of link that does work

### ISSUE THREAD

**Issue by erikvansebille** [OceanParcels/parcels#1417]
Since this morning, the builds on ReadTheDocs are failing, see e.g. https://readthedocs.org/projects/parcels/builds/21709578/. This seems to be because of this line in the log
```
/home/docs/checkouts/readthedocs.org/user_builds/parcels/checkouts/1416/docs/contributing.md:11: WARNING: local id not found in doc 'installation': 'installation-for-developers' [myst.xref_missing]
```

Associated with this line: 
https://github.com/OceanParcels/parcels/blob/fb7be59c8ce19d72120727dd766f0a50a70456a7/docs/contributing.md?plain=1#L11C139-L11C183

However, the link to `installation.rst#installation-for-developers` in `contributing.md` _does_ seem to work (and did work before). Not sure what is going on...

**Comment by erikvansebille** [comment]
Do you happen to know what's going on, @VeckoTheGecko?

**Comment by VeckoTheGecko** [comment]
The same workflow can be run locally using `sphinx-build -b linkcheck docs docs/_build`.

I think `(examples/documentation_MPI: line   20) broken    http://oceanparcels.org/#parallel_install - Anchor 'parallel_install' not found` is the actual culprit for the failing build (see #1418 for tracking on that. I was a bit trigger happy creating a new issue, but oh well 😅).

From what I can tell, the thing you flagged is just a warning so shouldn't fail things. An obvious fix hasn't jumped at me yet, but will keep looking.

**Comment by erikvansebille** [comment]
Thanks, now fixed in #1419
